backend/src/near.js

```diff
@@ -41,6 +41,35 @@ const getCurrentNodes = (nodes) => {
   return currentValidators;
 };
 
+const testnetConfig = {
+  nodeUrl: "https://rpc.testnet.nearprotocol.com",
+  networkId: "testnet",
+  contractName: "transfer-vote.near",
+};
+const mainnetConfig = {
+  nodeUrl: "https://rpc.mainnet.nearprotocol.com",
+  networkId: "mainnet",
+  contractName: "vote.f863973.m0",
```

Comment:
you had it backwards. For mainnet the contractName is `transfer-vote.near` whereas for testnet it is `vote.f863973.m0`. Can you test that with the config you put down, you can successfully query the contract state?